Library/ViewModels/CommentsViewModel.swift

```diff
@@ -39,6 +39,9 @@ public protocol CommentsViewModelOutputs {
 
   /// Emits a list of `Comment`s and the `Project` to load into the data source.
   var loadCommentsAndProjectIntoDataSource: Signal<([Comment], Project), Never> { get }
+
+  /// Emits a boolean that determines if the loader more activity indicator should show
+  var shouldShowLoadMoreIndicator: Signal<Bool, Never> { get }
```

Comment:
I think `shouldShowIndicator` is sufficient.

Reply:
Also update the documentation to:
"Emits a Bool that determines if the activity indicator should render."

Reply:
Isn't `shouldShowIndicator` a bit abstract since it doesn't say exactly what the indicator is about?

Reply:
The way I was looking at it was that there is no other "indicator" in the file but if you want to be slightly more descriptive you can do: `shouldShowActivityIndicator`

Reply:
I think we can leave it at `shouldShowIndicator `, we might not be able to use `shouldShowRefreshControl ` because it's not for refresh, but a load more.

Reply:
Fair enough although you make a good point that it does not say what exactly the indicator is doing so something like `shouldShowLoadingIndicator` is half way between what you and I are suggesting.